### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by barbaraliau**
- Adds additional description for what Kliento is and its limited use
- Adds note to prefer ContractKit over Kliento

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Kliento 
 
-Kliento (/client in esperanto/) is a set of client libraries for Celo in golang
+Kliento (/client in esperanto/) is a set of client libraries for Celo in Golang. It is a ported library from [ContractKit](https://github.com/celo-org/celo-monorepo/tree/master/packages/contractkit) and only supports a subset of what ContractKit does.
```
Comment (yorhodes) [inline README.md:3]: I realized it does provide a few additional features like block height query

**Review by yorhodes — APPROVED** [review]
lgtm with 1 comment

File: README.md (hunk 1 above)
Comment (barbaraliau) [reply README.md:3]: Oh cool. I'll open an issue to add to the README a short summary of features Kliento supports.

**Comment by barbaraliau** [comment]
@yorhodes quick question: should I merge, squash, or rebase PRs?

**Pull request merged**